Handle DB errors in tags search plugin

alikon opened the pull request:
updated to handle DB errors like #6710, #6711, #6632 and for #6591

Testing info
simulate a db error

alikon committed:
Handle DB errors in tags search plugin

updated to handle DB errors like #6710, #6711, #6632 and for #6591

Testing info
simulate a db error

nonumber commented:
@test Success

nonumber commented:
Not sure if this should get merged with regards to discussion on https://github.com/joomla/joomla-cms/pull/6710

alikon committed:
added the generic error msg

used the generic message string "There has been an error" as suggested in #6710 the string was added in #7073
